app/mailers/jobapps_mailer.rb

```diff
@@ -15,6 +15,13 @@ def application_denial(application_record)
          reply_to: reply_to
   end
 
+  def application_notification(subscription, position, applicant)
+    @position = position
```

Comment:
Since the instance variable stuff is so boilerplate in mailer methods, I've often gone for parallel assignment:

```ruby
@position, @applicant = position, applicant
```

but I'll leave that one to your discretion.